ioctl/newcmd/action/actionhash.go

```diff
@@ -22,6 +21,7 @@ import (
 	"github.com/spf13/cobra"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
+	protoV1 "google.golang.org/protobuf/proto"
```

Comment:
looks like error below `MarshalTextString` is a V1 function
can we find the corresponding func, so we can discard V1, and also use `"google.golang.org/protobuf/proto"` here?